### DISCUSSION TITLE
Sync failed with self hosted Nextcloud

### DISCUSSION THREAD

**Discussion by TheBoneZone** [floccusaddon/floccus#1265] [Q&A] [answered]
Hi,
first i am aware, thats not a floccus problem. But i hope someone runs in the same problem and have a solution.

I am working under Windows 10 and hosting a own Nextcloud Instance from Linuxserver.io under Ubuntu Server.  The installation provides a self signed SSL Certificate. I am using that instance only at private home network. Using a official SSL Certificate is not an option. 
I have worked through the Tutorial described here at the Wiki and tried serveral other ways. But the problem still persists.

Syncing Bookmarks fails with a Network error as long as i accepts the "NET::ERR_CERT_COMMON_NAME_INVALID" error after calling the Nextcloud Loginpage.

I have inserted the given self signed certificate in all available Windows 10 certificate stores. (own certificates, trusted root ca, etc.)

Maybe it is a Browser setting which i oversee? But searching the web brings up no hint. I really appreciate any hint.

**Comment by marcelklehr** [comment, marked as answer]
Opening the nextcloud domain in a normal browser session in the same browser and then clicking 'accpet risk' might allow floccus to connect as well. Beyond that I can't help you.
